angr/analyses/reaching_definitions/external_codeloc.py

```diff
@@ -4,10 +4,20 @@
 
 class ExternalCodeLocation(CodeLocation):
 
-    __slots__ = ()
+    __slots__ = ("call_string", )
 
-    def __init__(self):
+    def __init__(self, call_string=[None]):
```

Comment:
Don't use lists, sets, or dicts as the default value of arguments. Use `None` instead.